README.md

```diff
@@ -75,14 +75,16 @@ Build and run docker:
 
 ```
 docker build . -t noray
-docker run -p 8090:8090 -p 8091:8091 --env-file=.env -t noray
+docker run -p 8890:8890 -p 8890:8890/udp -p 8091:8091 --env-file=.env -t noray
```

Comment:
Based on default config, noray listens on 8890 TCP for commands, 8891 UDP for registrar, and 8891 HTTP for exposing metrics.

```suggestion
docker run -p 8890:8890 -p 8891:8891/udp -p 8891:8891 --env-file=.env -t noray
```